portfolio/src/main/webapp/template.html

```diff
@@ -34,16 +34,30 @@
     <!-- ******HEADER****** --> 
     <header class="header">
         <div class="container clearfix">                       
-            <img class="profile-image img-fluid float-left rounded-circle" style="width:200px;height:200px;" src="assets/images/headshot.jpg" alt="profile image" />
+            <img
+                class="profile-image img-fluid float-left rounded-circle" 
+                style="width:200px;height:200px;" src="assets/images/headshot.jpg" 
```

Comment:
Nit: inline styles are hard to override, could probably use existing class and the existing css file.